.pre-commit-config.yaml

```diff
@@ -13,3 +14,17 @@ repos:
         additional_dependencies:
           - "types-pytz"
           - "types-requests"
+        stages: [manual]
+
+  - repo: https://github.com/psf/black
+    rev: 22.3.0
+    hooks:
+      - id: black
+        args:
+          - "--line-length=120"
+
+  - repo: https://github.com/pycqa/isort
+    rev: 5.6.4
+    hooks:
+      - id: isort
+        args: [ "--profile", "black", "--filter-files" ]
```

Comment:
Why are these args required? I've used the `isort` pre-commit successfully with a number of projects which haven't required any additional arguments.

Reply:
The `filter-files` can be omitted. I think it's just there to not change any unmodified files, even if explicitly asked.

The `--profile "black"` actually formats the imports differently

with the black profile

```
from trino.exceptions import (
    DatabaseError,
    DataError,
    Error,
    IntegrityError,
    InterfaceError,
    InternalError,
    NotSupportedError,
    OperationalError,
    ProgrammingError,
    Warning,
)
```

without the profile

```
from trino.exceptions import (DatabaseError, DataError, Error, IntegrityError,
                              InterfaceError, InternalError, NotSupportedError,
                              OperationalError, ProgrammingError, Warning)
```

I personally like the black profile more as it is faster to scan.